### ISSUE TITLE
Read-only folders: Can't access settings when non-existent folder path is used

### ISSUE THREAD

**Issue by ConjuringCoffee** [SAP/abap-cleaner#160]
Hi Jörg-Michael, the new read-only folders already proved very useful. Unfortunately, if you add a folder path that doesn't exist, you get the following error later when interacting with the ABAP Cleaner:

![image](https://github.com/SAP/abap-cleaner/assets/72548231/a962ee32-1f14-4493-8fcf-a58b34d78e6b)

![image](https://github.com/SAP/abap-cleaner/assets/72548231/dc54c18c-2420-4005-b3b4-3388eb6002a1)

This error prevents the user from entering their settings to remove the folder path again. Editing the `main.cjf` file directly proved successful as a workaround.

```
  "readOnlyProfileName1": "Test",
  "readOnlyProfileDir1": "oops\\"
```

**Comment by jmgrassau** [comment]
oops!

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

this issue will be fixed in the next release. Always check for null…

Kind regards,
Jörg-Michael

**Comment by jmgrassau** [comment]
Hi ConjuringCoffee, 

thanks for this bug report – this should now be fixed in version 1.8.0!

Kind regards,
Jörg-Michael

**Comment by ConjuringCoffee** [comment]
Thank you 🙂
